README.md

```diff
@@ -148,6 +148,22 @@ apt::source { 'puppetlabs':
 },
 ~~~
 
+### Add an Apt source served over HTTPS
+
+~~~puppet
+include apt::transport_https
+
+apt::source { 'nodesource':
+  location => "https://deb.nodesource.com/node_0.10",
+  release  => $::lsbdistcodename,
+  repos    => 'main',
+  key      => {
+    'id'     => '9FD3B784BC1C6FC31A8A0A1C1655A0AB68576280',
+    'source' => 'https://deb.nodesource.com/gpgkey/nodesource.gpg.key',
+  }
+}
```

Comment:
This is pretty much what this is intended to fix. The NodeJS module has pretty much the code you suggested, @hunner, which causes a problem when you want to `ensure=>latest` for those packages.